frontend-ssr/services/Constants.ts

```diff
@@ -4,4 +4,4 @@ enum ChapterType {
   JS = 'js',
 }
 
-export { ChapterType as default };
+export default ChapterType;
```

Comment:
No default export in constants. We'll add many more things here so ;let's export them separately.

Reply:
Also, since you're at it please rename to lowercase - `constants.ts` :D